pkg/webhookconfig/policy.go

```diff
@@ -24,16 +24,19 @@ func (wrc *Register) contructPolicyValidatingWebhookConfig(caData []byte) *admre
 				caData,
 				true,
 				wrc.timeoutSeconds,
-				[]string{"clusterpolicies/*", "policies/*"},
-				"kyverno.io",
-				"v1",
+				admregapi.Rule{
+					Resources:   []string{"clusterpolicies/*", "policies/*"},
+					APIGroups:   []string{"kyverno.io"},
+					APIVersions: []string{"v1"},
+				},
 				[]admregapi.OperationType{admregapi.Create, admregapi.Update},
+				admregapi.Ignore,
```

Comment:
should this be set to `ignore` or `fail`?

Reply:
No, k8s allows `Ignore` or `Fail`.
```
   failurePolicy	<string>
     FailurePolicy defines how unrecognized errors from the admission endpoint
     are handled - allowed values are Ignore or Fail. Defaults to Fail.
```